### PR TITLE
Update canvas to be pixel perfect

### PR DESCRIPTION
**Pull request by RUSshy**
Useful when zooming

### CODE REVIEW

File: src/app/layout/WorkSpace/modules/PackCanvas/PackCanvas.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ const useStyles = makeStyles(({ bgPixel }) =>
       marginTop: (props: StyleProps) => `${props.height / -2}px`,
       transform: (props: StyleProps) =>
         `scale(${props.scale}) translate(${props.offsetX}px,${props.offsetY}px)`,
+      image-rendering: pixelated,
```
Comment (SilenceLeo) [inline src/app/layout/WorkSpace/modules/PackCanvas/PackCanvas.tsx:43]: CSS in JS needs to use `imageRendering: 'pixelated',` .

Comment (RUSshy) [reply src/app/layout/WorkSpace/modules/PackCanvas/PackCanvas.tsx:43]: oh sorry, i am not good with CSS and JS

Comment (SilenceLeo) [reply src/app/layout/WorkSpace/modules/PackCanvas/PackCanvas.tsx:43]: @RUSshy
Thank you for your PR, I have commit this optimization.   https://github.com/SilenceLeo/snowb-bmf/commit/29e831c4b2f346ab7024a447a6b9dd9b308d6508

**Comment by SilenceLeo** [comment]
https://github.com/SilenceLeo/snowb-bmf/commit/29e831c4b2f346ab7024a447a6b9dd9b308d6508

**Pull request closed without merging**